components/clubs-page.jsx

```diff
@@ -105,9 +106,12 @@ var BottomCTA = React.createClass({
     return(
       <div className="row">
         <div className="col-sm-offset-2 col-sm-8 col-md-offset-2 col-md-8 col-lg-offset-2 col-lg-8">
-            <img className="divider" src="/img/clubs-line-divider.svg" alt="line divider" />
-          <PageEndCTA linkTo={this.props.ctaLink} ctaBtnText="Add your club to the map">
-            <p>Do you meet regularly with a group of learners to increase web literacy skills?</p>
+          <PageEndCTA linkTo={this.props.ctaLink}>
+            <div>
+              <img className="divider" src="/img/clubs-line-divider.svg" alt="line divider" />
+              <p>Do you meet regularly with a group of learners to increase web literacy skills?</p>
+              <a className="btn btn-awsm" onClick={this.props.onClick} data-modal-name="ModalAddYourClub">Add your club to the map</a>
```

Comment:
Since we're not using `data-modal-name` anymore can you get rid of this?

Other than that, this looks great, feel free to merge when Travis is happy!